docs/content/tutorials/tutorial-dapr/index.md

```diff
@@ -20,16 +20,33 @@ For more details on the app and access to the source code, visit the `tutorials/
 
 - [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
 - [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
-- [Radius environment]({{< ref "installation#step-3-initialize-radius" >}})
+- [Setup a supported Kubernetes cluster](https://docs.radapp.io/guides/operations/kubernetes/overview/#supported-clusters)
 - [Dapr installed on your Kubernetes cluster](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/)
 
-## Step 1: Define the application, `backend` container, and Dapr state store
+## Step 1: Initialize a Radius Environment
+
+1. Begin in a new directory for your application:
+
+   ```bash
+   mkdir rad-dapr
```

Comment:
This conflicts with https://github.com/radius-project/samples/pull/654/files#r1372333903. We should have the directory match the app name

Reply:
fixed